src/ui/common/src/reducers/nodeSelection.ts

```diff
@@ -29,9 +30,15 @@ export const OperatorTypeToNodeTypeMap: { [key in OperatorType]: NodeType } = {
 
 export const ArtifactTypeToNodeTypeMap: { [key in ArtifactType]: NodeType } = {
   [ArtifactType.Table]: NodeType.TableArtifact,
-  [ArtifactType.Float]: NodeType.FloatArtifact,
+  [ArtifactType.Numeric]: NodeType.NumericArtifact,
   [ArtifactType.Bool]: NodeType.BoolArtifact,
   [ArtifactType.Json]: NodeType.JsonArtifact,
+  [ArtifactType.String]: NodeType.GenericArtifact,
+  [ArtifactType.Dict]: NodeType.GenericArtifact,
+  [ArtifactType.Tuple]: NodeType.GenericArtifact,
+  [ArtifactType.Image]: NodeType.GenericArtifact,
+  [ArtifactType.Bytes]: NodeType.GenericArtifact,
+  [ArtifactType.Picklable]: NodeType.GenericArtifact,
```

Comment:
I forgot, do we plan to keep all these as GenericArtifacts in the foreseeable future?

Reply:
No, after this PR, we will implement custom types for each type, so that when for example we get an image type, we can show the image preview on the UI.